README.md

```diff
@@ -11,3 +11,66 @@ This repository contains a collection of Helm charts curated by Mozilla's Servic
+voice                voice-prod                      deployed   failed to upgrade chart for release [voice-prod]: YAML parse error on mozilla-common-voice/templates/configmap.yaml: error converting YAML to JSON: yaml: line 17: did not find expected key   7d19h
+```
+Here we can clearly see an error, we are missing a key in our configmap. Running `kubectl describe helmrelease voice-prod` will give you even more information.
+
+### Using logs for debugging
+#### Flux
+The same information found out above can be found in the logs of the 2 components involved: flux and helm. First of all we will check the logs of Flux to see what is in there.
+Running `kubectl logs -l=app=flux -n fluxcd` you will get the logs of Flux where we can make sure that it correctly detected the change in the Chart and will try to apply it.
+```
+kubectl logs -l=app=flux -n fluxcd
+ts=2020-04-17T09:24:41.796293402Z caller=loop.go:133 component=sync-loop event=refreshed url=ssh://git@github.com/mozilla-it/voice-infra branch=master HEAD=26626653ecca80f3d43f6e42aa3376af15755622
+```
+Nothing wrong here, moving on and checking helm-operator logs
+
+#### Helm
+helm-operator logs are also a good resource of information when debugging. Check them running `kubectl logs -f -l=app=helm-operator -n fluxcd`. Here we will see the same error that we saw when describing the HelmRelease:
+```
+kubectl logs -f -l=app=helm-operator -n fluxcd                                                                   :(
+ts=2020-04-17T09:24:37.559875402Z caller=operator.go:307 component=operator info="enqueuing release" resource=voice-prod:helmrelease/voice
+ts=2020-04-17T09:24:37.647535928Z caller=release.go:353 component=release release=voice-prod targetNamespace=voice-prod resource=voice-prod:helmrelease/voice helmVersion=v3 info="release has not yet been processed" action=upgrade
+ts=2020-04-17T09:24:37.663182157Z caller=helm.go:69 component=helm version=v3 info="preparing upgrade for voice-prod" targetNamespace=voice-prod release=voice-prod
+ts=2020-04-17T09:24:37.790261829Z caller=release.go:216 component=release release=voice-prod targetNamespace=voice-prod resource=voice-prod:helmrelease/voice helmVersion=v3 error="Helm release failed" revision=0.1.13 err="failed to upgrade chart for release [voice-prod]: YAML parse error on mozilla-common-voice/templates/configmap.yaml: error converting YAML to JSON: yaml: line 17: did not find expected key"
+```
+
+### Fixing the problem
+We now know where the problem is, a missing or bad value which makes not possible rendering the configmap. Here one of the options you have is to go an verify if you are missing some value, inc ase you are not able to find it, other option is to force localy the render of the chart. This will allow us to reproduce the error locally and fix it faster.
```

Comment:
We now know where the problem is, a missing or bad value which makes it impossible to render the configmap. Here one of the options you have is to go and verify if you are missing some value, in case you are not able to find it.  The other option is to locally render the chart. This will allow us to reproduce the error locally and fix it faster.